curriculum/challenges/english/24-b1-english-for-developers/learn-present-perfect-while-talking-about-accessibility/671a4e181afe602d37786dd8.md

```diff
@@ -0,0 +1,76 @@
+---
+id: 671a4e181afe602d37786dd8
+title: Task 27
+challengeType: 22
+dashedName: task-27
+---
+
+<!-- REVIEW -->
+
+# --description--
+
+This is a review of the entire dialogue you just studied.
+
+# --instructions--
+
+Place the following phrases in the correct spot: `negative comments`, `We've been addressing`, `That’s a relief`, `make our app more accessible`, `keep making adjustments`.
+
+# --fillInTheBlank--
+
+## --sentence--
+
+`Tom: Hey Sophie, have you been working on the screen reader updates recently?`
+
+`Sophie: Yes, I started a few months ago. BLANK the feedback from our last user survey. How’s the design side coming along?`
+
+`Tom: Pretty well, actually. I've been tweaking the layouts based on the guidelines you shared.`
+
+`Sophie: That’s good to hear. Do you think the users have noticed the improvements?`
+
+`Tom: I think so. We haven’t been receiving as many BLANK.`
+
+`Sophie: BLANK. How long do you think we’ll need to BLANK?`
+
+`Tom: Probably a few more weeks. We've been making steady progress, and I believe we’re close to finalizing the major changes.`
+
+`Sophie: That’s good news! It feels great to BLANK.`
```

Comment:
````suggestion
```md
Tom: Hey Sophie, have you been working on the screen reader updates recently?

Sophie: Yes, I started a few months ago. BLANK the feedback from our last user survey. How’s the design side coming along?

Tom: Pretty well, actually. I've been tweaking the layouts based on the guidelines you shared.

Sophie: That’s good to hear. Do you think the users have noticed the improvements?

Tom: I think so. We haven’t been receiving as many BLANK.

Sophie: BLANK. How long do you think we’ll need to BLANK?

Tom: Probably a few more weeks. We've been making steady progress, and I believe we’re close to finalizing the major changes.

Sophie: That’s good news! It feels great to BLANK.
```
````

Reply:
Hey @naomi-lgbt I've just tried this change you suggested on another PR I'm working and it would look like this: 
![image](https://github.com/user-attachments/assets/28499394-4c53-4a41-bd0b-de13f0ba3ae5)

It wouldn't work well for reading the text. Do you have any other ideas on formatting this part of the content?
I've left a comment on this PR with more details. Also, if you can, I'd love to meet and try to find a better solution.